controllers/userController.ts

```diff
@@ -80,5 +86,34 @@ const deleteUser = async (req: Request, res: Response) => {
     throw new Error('User not found')
   }
 }
+// @desc    Verify user jwt
+// @route   POST /api/users/verifyJWT
+const verifyUserJWT = async (req: Request, res: Response) => {
+  console.log(req.body.jwt)
+  var token = req.body.jwt
+  if (!token) return res.status(401).send({ auth: false, message: 'No token provided.' });
+
+  jwt.verify(token, config.jwtSecret, async function (err: Error, decoded: any) {
```

Comment:
Instead of any u can use this interface, { id: string; iat: number; exp: number }